build.gradle

```diff
@@ -1,30 +1,51 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+import java.time.Instant
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
 plugins {
-    id 'org.springframework.boot' version '2.1.5.RELEASE'
     id 'java'
+    id 'org.springframework.boot' version '2.1.9.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
 }
 
-apply plugin: 'io.spring.dependency-management'
+repositories {
+    mavenCentral()
+}
 
-group = 'uk.gov.justice.digital.probation.court.list'
-version = '0.0.1-SNAPSHOT'
 project.sourceCompatibility = 11
 project.targetCompatibility = 11
 
+group = 'uk.gov.justice.probation'
+
+def today = Instant.now()
+def todayDate = LocalDate.now()
+def formatter = DateTimeFormatter.ofPattern('yyyy-MM-dd')
+version = todayDate.format(formatter)
+
 configurations {
     compileOnly {
         extendsFrom annotationProcessor
     }
 }
 
-repositories {
-    mavenCentral()
+jar {
+    baseName = rootProject.name
+    manifest {
+        attributes (
+            'Implementation-Title': rootProject.name,
+            'Implementation-Version': System.getenv('CI') ? "${version}.${System.getenv('CIRCLE_BUILD_NUM')}" : version
+        )
+    }
 }
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-security'
     implementation 'org.springframework.security.oauth:spring-security-oauth2:2.3.6.RELEASE'
     implementation 'org.springframework.security:spring-security-jwt:1.0.10.RELEASE'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
```

Comment:
two more dependencies worth thinking about - applicationinsights for monitoring logs / requests etc and graceful shutdown to drain connection pools before shutting down

Reply:
I was planning to add in app insights later, I need to sort out an account for Probation. 
 I've created a ticket for it